src/platform/K32W/BLEManagerImpl.cpp

```diff
@@ -805,17 +833,16 @@ CHIP_ERROR BLEManagerImpl::ConfigureAdvertisingData(void)
     scanRsp.aAdStructures = scan_rsp_data;
 
     /**************** Prepare advertising parameters *************************************/
-    if ((NumConnections() == 0 && !ConfigurationMgr().IsPairedToAccount()) || mFlags.Has(Flags::kFastAdvertisingEnabled))
+    if (mFlags.Has(Flags::kFastAdvertisingEnabled))
     {
-        adv_params.minInterval = CHIP_DEVICE_CONFIG_BLE_FAST_ADVERTISING_INTERVAL_MIN;
```

Comment:
Do you think you could use the `_MIN`, `_MAX` constants which are used across all platforms now? Note that they were set to match the spec, so I'm not sure you need to override it for your platform, do you? :)

Reply:
Agree, replaced the constants with the ones defined in common CHIP codebase.